src/main/java/frc/robot/subsystems/flywheel/Flywheel.java

```diff
@@ -0,0 +1,47 @@
+package frc.robot.subsystems.flywheel;
+
+import static frc.robot.subsystems.flywheel.FlywheelConstants.*;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.filter.SlewRateLimiter;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.lib.logfields.LogFieldsTable;
+import frc.robot.Robot;
+import frc.robot.subsystems.flywheel.io.FlywheelIO;
+import frc.robot.subsystems.flywheel.io.FlywheelIOSim;
+import frc.robot.subsystems.flywheel.io.FlywheelIOSparkMax;
+
+public class Flywheel extends SubsystemBase {
+    private final LogFieldsTable fieldsTable = new LogFieldsTable(getName());
+    private final FlywheelIO io = Robot.isSimulation()
+            ? new FlywheelIOSim(fieldsTable)
+            : new FlywheelIOSparkMax(fieldsTable);
+
+    private final SlewRateLimiter upperRollerSpeedAccelerationLimiter = new SlewRateLimiter(
+            MAX_ACCELERATION_VOLTS_PER_SECOND);
+    private final SlewRateLimiter lowerRollerSpeedAccelerationLimiter = new SlewRateLimiter(
+            MAX_ACCELERATION_VOLTS_PER_SECOND);
+
+    public Flywheel() {
+        fieldsTable.update();
+    }
+
+    @Override
+    public void periodic() {
+    }
+
+    public void setSpeed(double upperRollerDemandVoltage, double lowerRollerDemandVoltage) {
+        upperRollerDemandVoltage = upperRollerSpeedAccelerationLimiter.calculate(upperRollerDemandVoltage);
```

Comment:
Why not making it two different functions and then calling them? Seems to me more elegant

Reply:
That what I did at the beginning, but then I was like "when will you call only one of them"? I don't know, makes it harder to forget to call both of them. Maybe will change